modulegen/_template/examples_test.go.tmpl

```diff
@@ -0,0 +1,36 @@
+{{ $entrypoint := Entrypoint }}{{ $lower := ToLower }}{{ $title := Title }}package {{ $lower }}_test
```

Comment:
The examples will live in the `_test` package so that the module has to be imported.